src/broadcast_stage.rs

```diff
@@ -140,7 +142,7 @@ impl BroadcastStage {
         let mut receive_index = entry_height;
         let me = crdt.read().unwrap().my_data().clone();
         loop {
-            let broadcast_table = crdt.read().unwrap().compute_broadcast_table();
+            let broadcast_table = voting_nodes.read().unwrap().compute_broadcast_table(crdt);
```

Comment:
awesome!  this is what i was thinking.  "liveness" is from the ledger, but network info is via gossip